cmd/agent/main.go

```diff
@@ -79,7 +85,13 @@ See https://kairos.io/docs/upgrade/manual/ for documentation.
 			if c.Args().Len() == 1 {
 				v = c.Args().First()
 			}
-			return agent.Upgrade(v, c.String("image"), c.Bool("force"), c.Bool("debug"), c.Bool("strict-validation"), configScanDir)
+
+			return agent.Upgrade(
```

Comment:
I'm wondering, since we are in control of the pull on the elemental side, if we should just use ENV vars as those are secrets

Reply:
good idea, but Im not sure those are loaded via env vars...let me test that

Reply:
ah no, this is not loaded via env vars becuase its not part of the Config struct, its just flags :/